README.md

```diff
@@ -8,26 +9,59 @@ These tutorials will get you up to speed and sending messages with Solace techno
 
 ## Contents
 
-This repository contains code and matching tutorial walk throughs for five different basic Solace messaging patterns. For a nice introduction to the Solace API and associated tutorials, check out the [tutorials home page](https://solacesamples.github.io/solace-samples-nodejs/).
+This repository contains:
+
+* Code and matching tutorial walk throughs for five different **basic** Solace messaging patterns. For a nice introduction to the Solace API and associated tutorials, check out the [tutorials home page](https://solacesamples.github.io/solace-samples-nodejs/).
+
+    See the individual tutorials for details:
+
+    - [Publish/Subscribe](https://solacesamples.github.io/solace-samples-nodejs/publish-subscribe): Learn how to set up pub/sub messaging on a Solace VMR.
+    - [Persistence](https://solacesamples.github.io/solace-samples-nodejs/persistence-with-queues): Learn how to set up persistence for guaranteed delivery.
+    - [Request/Reply](https://solacesamples.github.io/solace-samples-nodejs/request-reply): Learn how to set up request/reply messaging.
+    - [Confirmed Delivery](https://solacesamples.github.io/solace-samples-nodejs/confirmed-delivery): Learn how to confirm that your messages are received by a Solace message router.
+    - [Topic to Queue Mapping](https://solacesamples.github.io/solace-samples-nodejs/topic-to-queue-mapping): Learn how to map existing topics to Solace queues.
+
+* Additional sample code, showing how to make use of **advanced** features of the Solace message router is available in the [advanced-samples directory](https://github.com/SolaceSamples/solace-samples-nodejs/tree/master/src/advanced-samples).
 
-## Checking out and Building
+    - [Secure Session](https://github.com/SolaceSamples/solace-samples-nodejs/tree/master/src/advanced-samples): Learn how to use secure connection to the server, and server and client certificate authentication.
```

Comment:
These links are not to the actual tutorials. Should be updated to point at the correct target.